bindings/netstandard/ElectionGuard/ElectionGuard.Decryption/ElectionRecord/ElectionRecordManager.cs

```diff
@@ -0,0 +1,305 @@
+            var json = await File.ReadAllTextAsync(file);
+            var device = new EncryptionDevice(json);
+            devices.Add(device);
+        }
+
+        return devices;
+    }
+
+    private static async Task<List<CiphertextBallot>> ImportBallotsAsync(string ballotFolder)
+    {
+        var ballots = new List<CiphertextBallot>();
+
+        foreach (var file in Directory.EnumerateFiles(ballotFolder))
+        {
+            var json = await File.ReadAllTextAsync(file);
+            var ballot = new CiphertextBallot(json);
+            ballots.Add(ballot);
+        }
+
+        return ballots;
+    }
+
+    private static async Task<List<PlaintextTallyBallot>> ImportChallengedBallotsAsync(string challengedFolder)
+    {
+        var ballots = new List<PlaintextTallyBallot>();
+
+        foreach (var file in Directory.EnumerateFiles(challengedFolder))
+        {
+
+            var ballot = await ImportAsync<PlaintextTallyBallot>(file);
+            ballots.Add(ballot);
+        }
+
+        return ballots;
+    }
+
+    private static async Task<PlaintextTally> ImportTallyAsync(string jsonFile)
+    {
+        return await ImportAsync<PlaintextTally>(jsonFile);
+    }
+
+    private static async Task<CiphertextTallyRecord> ImportEncryptedTallyAsync(string jsonFile)
+    {
+        return await ImportAsync<CiphertextTallyRecord>(jsonFile);
+    }
+
+    private static async Task<T> ImportAsync<T>(string jsonFile)
+    {
+        var json = await File.ReadAllTextAsync(jsonFile);
+        return JsonConvert.DeserializeObject<T>(json, SerializationSettings.NewtonsoftSettings())!;
+    }
+
+    private static async Task ExportParts(ElectionRecordData record, string tempFolder)
+    {
+        // export the guardians
+        await ExportGuardiansAsync(Path.Combine(tempFolder, GUARDIAN_FOLDER), record.Guardians!);
+
+        // export the top level
+        await ExportSummaryAsync(tempFolder, record.Constants, record.Manifest, record.Context);
+
+        // export the devices
+        await ExportDevices(Path.Combine(tempFolder, DEVICE_FOLDER), record.Devices!);
+
+        // export the ballots
+        await ExportBallotsAsync(Path.Combine(tempFolder, BALLOT_FOLDER), record.EncryptedBallots);
+
+        // export the challenged ballots v2 / spoiled ballots v1
+        await ExportChallengedBallotsAsync(Path.Combine(tempFolder, CHALLENGED_FOLDER), record.ChallengedBallots);
+
+        await ExportEncryptedTallyAsync(Path.Combine(tempFolder, ENCRYPTED_TALLY_FILENAME), record.EncryptedTally);
+
+        await ExportTallyAsync(Path.Combine(tempFolder, TALLY_FILENAME), record.Tally);
+    }
+
+    private static async Task ExportSummaryAsync(string tempFolder, ElectionConstants constants, Manifest manifest, CiphertextElectionContext context)
+    {
+        await File.WriteAllTextAsync(Path.Combine(tempFolder, CONSTANTS_FILENAME), JsonConvert.SerializeObject(
+            constants,
+            SerializationSettings.NewtonsoftSettings()));
```

Comment:
The UI app sets this up

        JsonConvert.DefaultSettings = SerializationSettings.NewtonsoftSettings;

so every call gets the serialization settings without having to add it to each one in case it is forgotten.  The CLI could do the same thing so that it was not needed in each of these calls.